tests/clients/github/test_github.py

```diff
@@ -1,10 +1,12 @@
 from gevent import sleep
 from os import getenv
+import pytest
 
 from jumpscale.loader import j
 from tests.base_tests import BaseTests
 
 
+@pytest.mark.skip()
```

Comment:
let's make these tests run nightly instead of skipping them
```suggestion
@pytest.mark.integration()
```